cmd/notation/verify.go

```diff
@@ -100,7 +98,8 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 	// core verify process.
 	_, outcomes, err := notation.Verify(ctx, verifier, repo, verifyOpts)
 	if err != nil {
-		logger.Error(err)
+		// print out verification error
+		fmt.Println("Verification error:", err)
```

Comment:
Will this print error message for all signatures associated with artifact? If so we shouldn't print it by default.

Reply:
No, it won't. This printout only prints errors generated one level above each signature's verification. For example: error: "the artifact has no applicable trust policy". This PR intends to resolve this [issue](https://github.com/notaryproject/notation/issues/477).